src/converters/helpers.ts

```diff
@@ -22,14 +23,112 @@ export function nullResult(...messages: Message[]): MemberResult {
   };
 }
 
-export function isCatalogMember(m: any): m is CatalogMember {
-  return is.string(m?.type) && is.string(m?.name);
+export function isCatalogMember(m: any, partial = false): m is CatalogMember {
+  return is.string(m?.type) && (is.string(m?.name) || partial);
 }
 
-export const catalogMemberProps = ["description", "info"];
+export const catalogMemberProps: CopyProps[] = [
+  "id",
+  "description",
+  "info",
+  "infoSectionOrder",
+  "shortReport",
+  {
+    v7: "shortReportSections",
+    v8: "shortReportSections",
+    translationFn: (srs: any[]) =>
+      srs.length > 0
+        ? srs.map((shortReport) => {
+            return {
+              name: shortReport.name,
+              content: shortReport.content,
+              show: shortReport.isOpen,
+            };
+          })
+        : undefined,
+  },
+  { v7: "isShown", v8: "show" },
+  "splitDirection",
+  "url",
+  {
+    v7: "opacity",
+    v8: "opacity",
+    translationFn: (opacity: any) =>
+      is.string(opacity)
+        ? parseFloat(opacity)
+        : is.number(opacity)
+        ? opacity
+        : undefined,
```

Comment:
This should probably try `parseFloat(opacity)` and use that unless it's NaN in which case it should not be set.